crates/nargo/Cargo.toml

```diff
@@ -11,6 +11,10 @@ edition.workspace = true
 rustc_version = "0.4.0"
 
 [dependencies]
+
+# Backends
+acvm-backend-barretenberg = { path = "../acvm_backend_barretenberg" }
```

Comment:
nargo core is meant to be independent of its environment so linking it to the CLI like this isn't correct. We should remove the prove/verify operations from this package in the meantime.

Reply:
yep good point